module-2/backend/week_two.md

```diff
@@ -5,37 +5,118 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+
+  A: ActiveRecord is an ORM that allows you to manipulate a SQL database more easily.
+  
 2. What is a migration?
+
+  A: A migration updates your schema with the changes that you want to make to your database structure.
+    
 3. How does a table relate to a model?
+
+  A: A model represents a table in a database.
+    
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+
+  A: Class method - e.g. belongs_to :tasks.
+  
 5. What do they allow you to do?
+
+  A: They connect tables in a database through a foreign key.
+  
 6. What's the difference between agile workflow and waterfall method?
+
+  A: Agile workflow breaks the work up into chunks/iterations, tests and refactors each iteration along the way. Waterfall method does each piece of work separately (design, code, test, etc.) one after another.
+  
 7. What is the difference between `#new` and `#create`?
+
+  A: "new" initializes an object but doesn't save, "create" initializes and saves.
+  
 8. At a basic level, what does cURL allow you to do?
+
+  A: It allows you to put information into a database through the command line.
```

Comment:
It makes HTTP requests from the command line. You're totally right that we can use those requests to put information into a database (like we did in Rush Hour).